modules/store.js

```diff
@@ -75,6 +75,15 @@ export const reducer = handleActions({
 
 }, initialState);
 
+export const reducer = function wrappedReducer(state, action) {
+  // if state is undefined then it can't be converted to mutable
+  let mutableState = state;
+  if (mutableState !== undefined) {
+    mutableState = getMutableState(state);
+  }
+  return getImmutableState(reduxAsyncReducer(mutableState, action));
+};
+
```

Comment:
I will say I'm not the biggest fan of separate reducer exports because it adds extra api overhead to manage for the savings of 2 identity function calls, but I just pushed the change with documentation updated. Let me know what you think.